packages/shared/src/utils/config/platform-config.ts

```diff
@@ -9,7 +9,7 @@ type TPlatform = {
     is_hard_redirect: boolean;
     platform_name: string;
     route_to_path: string;
-    url: string;
+    url?: string;
```

Comment:
why are we making `url` optional?

Reply:
URL is an optional prop for the configuration